### PR TITLE
Reset status code for Koa (#677)

### PR DESCRIPTION
**Pull request by livoras**
fixed https://github.com/zeit/next.js/issues/677

### CODE REVIEW

File: server/render.js [hunk 1]
```diff
@@ -153,6 +153,7 @@ export async function serveStaticWithGzip (req, res, path) {
     await serveStatic(req, res, gzipPath)
   } catch (ex) {
     if (ex.code === 'ENOENT') {
+      res.statusCode = 200
```
Comment (arunoda) [inline server/render.js]: This seems like a legit thing to do.
But, I am fixing the root cause here in #640 
We'll take it pretty soon.

**Comment by arunoda** [comment]
We fixed the root cause of this issue with the PR: https://github.com/zeit/next.js/pull/704

**Comment by arunoda** [comment]
I think we don't need this fix.

**Comment by arunoda** [comment]
@livoras if the current master didn't fix your issue, let us know.

**Comment by aga5tya** [comment]
> @livoras if the current master didn't fix your issue, let us know.

@arunoda, the issue still exists on the latest master.
[zeit/next.js#677](https://github.com/zeit/next.js/issues/677#issuecomment-273266526)

**Pull request closed without merging**